stock_reception_screen/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock - Reception screen",
     "summary": "Dedicated screen to receive/scan goods.",
-    "version": "13.0.1.1.3",
+    "version": "13.0.1.1.4",
```

Comment:
do not change the version unless you need a specific migration step, the OCA bot will do it on merge :wink: